showcase/src/app/home/_home.component.scss

```diff
@@ -62,6 +55,27 @@
   }
 
   .about-background {
-    height: 230px;
+    height: 100px;
+    background-size: 150px 60px;
+  }
+}
+
+@media #{$tablet-query} {
+  .stark-app-bar-content-center {
+    align-self: center;
+    margin-bottom: 15px;
+  }
+}
+
+@media #{$mobile-only-query} {
+  .stark-app-bar-page-title {
+    top: 60px;
+    left: 0;
+    position: absolute;
+    max-width: 100%;
+    background-image: url(/assets/stark-core/logo/stark_logo_transparent.svg);
```

Comment:
Is this working? I don't think that this path is configured anywhere.... the right path should be the one you define in the `angular.json` file of the stark-core ;)

Reply:
It is working too